### PR TITLE
FIX file sync removes folders with dot in name

### PR DESCRIPTION
**Pull request by jonom**
/cc @tractorcow

### CODE REVIEW

File: filesystem/Folder.php [hunk 1]
```diff
@@ -165,6 +165,7 @@ public function syncChildren() {
 
 				// Check allowed extensions, unless admin users are allowed to bypass these exclusions
 				if($checkExtensions
+					&& !is_dir("$baseDir/$actualChild")
```
Comment (tractorcow) [inline filesystem/Folder.php]: On line 199 it does `$baseDir . $actualChild` to concatenate these. Does does $baseDir have a trailing slash?

Comment (jonom) [inline filesystem/Folder.php]: Yes looks like it always does after some quick testing. Have updated.

**Comment by tractorcow** [comment]
I'm going to forgive the test failure; It's due to imagick no longer being supported on 5.3.

**Comment by jonom** [comment]
Cheers

**Pull request merged**